docker/entrypoint

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# In case when mapped user id by `docker run -u` is not created inside docker image
+# The `$HOME` variable points to `/` - which prevents any tool to write to, as it requires root access
+# In such case we set `$HOME` to `/tmp` as it should r/w for everyone
+
+if [ "$HOME" = "/" ] ; then
+    export HOME=/tmp
+fi
+
+# In case of running as root, use `umask` to reduce problem of file permission on host
+if [ "$(id -g)" = "0" ] && [ "$(id -u)" = "0" ] ;
+then
+    umask 0000
+fi
+
+# Export this image specific Environment variables
+# Those variables are important to use dedicated folder for all cache and predefined config file
+export EMSDK=/emsdk_portable
+export EM_DATA=/emsdk_portable/.data
```

Comment:
EM_DATA is not known to emsdk or emscripten, I don't think we need it here.

Reply:
removed